lasio/reader.py

```diff
@@ -722,7 +726,14 @@ class SectionParser(object):
     """
 
     def __init__(self, title, version=1.2):
-        if title.upper().startswith("~C"):
+        las3_section_indicators = ['_DATA', '_PARAMETER', '_DEFINITION']
```

Comment:
For now, I think we should only force sections like these to be read explicitly as "LAS 3" sections **if** the parsed version of the file is actually 3.0. If it isn't, there's plenty of chance that for example a genuinely LAS 2 file might have these strings in the section heading for totally different sections.

It's a pedantic point that adds complexity, I know, but I don't want sections that are currently being parsed a certain way to change, at least temporarily. Does that make sense? We're probably not testing the situation I outline in regression tests now, so it would go unnoticed.

Reply:
Yep, It is a great point.  I think I can add a 3.0 version check to the if statement.

Reply:
Understood, it's a tricky one! Thanks for adjusting.